.github/workflows/examples/edd.yml

```diff
@@ -7,15 +7,12 @@ on:
   workflow_dispatch:
 
 env:
-  # The plugin slug
+  # The plugin slug (example: searchwp)
```

Comment:
Now that I think about it (sorry) SearchWP is a perfect example of why the slug isn't going to work. When we used the `searchwp` slug, we got version 3. V4 got a new slug (it's `searchwp-4` if you're curious). And that isn't the path we want this installed to, so both are going to be required.

Reply:
You're absolutely right; we already learned that lesson! Fixed.